src/google/signer.rs

```diff
@@ -213,6 +217,40 @@ impl Signer {
 
         Err(anyhow!("token not found"))
     }
+
+    fn canonicalize(
+        &self,
+        req: &impl SignableRequest,
+        method: SigningMethod,
+        cred: &Credential,
+    ) -> Result<CanonicalRequest> {
+        let mut creq = CanonicalRequest::new(
+            req,
+            method,
+            None,
+            SigningAlgorithm::Goog4Rsa,
+            "europe-west3".to_string(),
```

Comment:
Hardcoded？

Reply:
I am still testing things, sorry for the noise :)